com.github.iortcw.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: com.github.iortcw
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "20.08"
+command: iortcw_sp.sh
+
+finish-args:
+- --device=dri
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
```

Comment:
```suggestion
```

Reply:
Only if it actually works natively on Wayland, that is.

Reply:
Removed it